sympy/geometry/tests/test_line.py

```diff
@@ -159,6 +174,7 @@ def test_basic_properties_2d():
 
     assert Line((1, 1), slope=1) == Line((1, 1), (2, 2))
     assert Line((1, 1), slope=oo) == Line((1, 1), (1, 2))
+    assert Line((1, 1), slope=oo).bounds == (1, 1, 1, 2)
```

Comment:
The bounds should be `(1,-oo,1,oo)`; bounds could also be defined for Parabola as `(-oo,ymin,oo,ymax)` where ymin/max is the y value at the vertex when the parabola opens up/down and ymax/ymin is +/-oo.

Reply:
currently the bounds is identified as the bounding rectangle for the geometric figure and since the line with oo slope is defined as 1,1 to 1,2, shouldn't the bounds be also these?

Reply:
I understand the fact that the line with slope oo could be anything from 1,2 to 1,oo but i think that it is currently implemented as just getting the line to the next nearest point.

Reply:
Yes, it looks like Line respects the args given to construct it and the bounds are used for making plotting decisions as I recall. So perhaps this should remain unchanged.